run_dir/design/index.html

```diff
@@ -13,39 +13,39 @@ <h2> Get Access </h2>
 
 <p>Genomics status displays information from multiple data sources. <span id="updated-status"></span></p>
 
-<form class="form-horizontal" role="form">
+<form role="form">
     <div class="form-group form-group-lg statusdb-search" id="homepage-searchbar">
-        <label for="typeahead-input" class="control-label">Search for Project, Workset or Flow Cell:</label>
-        <input autofocus class="typeahead form-control input-lg" type="text" placeholder="Project, Workset or Flowcell Name / ID">
-        <span class="glyphicon glyphicon-refresh glyphicon-spin input-spinner subtle-spinner"></span>
+        <label for="typeahead-input" class="col-form-label fs-5 fw-bold">Search for Project, Workset or Flow Cell:</label>
+        <input autofocus class="typeahead form-control form-control-lg input-lg" type="text" placeholder="Project, Workset or Flowcell Name / ID">
+        <span class="fa fa-sync fa-spin input-spinner subtle-spinner"></span>
     </div>
 </form>
 
-<div class="row homepage-ctas">
+<div class="row homepage-ctas mt-3 justify-content-md-center">
 	<div class="col-lg-offset-3 col-lg-2 col-sm-4 col-xs-6">
 		<div class="well">
-			<h3><a href="/projects"><i class="glyphicon glyphicon-search"></i></a></h3>
+			<h3><a href="/projects"><i class="fa fa-search"></i></a></h3>
 			<p><a class="btn btn-primary" href="/projects">View All Projects</a></p>
```

Comment:
I think all three of thes buttons would look better with the `btn-lg` class.

Reply:
Done!